common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -71,22 +71,22 @@ class @Sequence
         when 'done' then element.addClass('progress-done')
 
   toggleArrows: =>
-    @$('.sequence-nav-buttons a').unbind('click')
+    @$('.sequence-nav-button').unbind('click')
 
     if @contents.length == 0
-      @$('.sequence-nav-buttons .prev a').addClass('disabled').attr('aria-hidden', 'true')
-      @$('.sequence-nav-buttons .next a').addClass('disabled').attr('aria-hidden', 'true')
+      @$('.sequence-nav-button.button-previous').addClass('disabled').attr('aria-hidden', 'true').attr('disabled', 'disabled')
+      @$('.sequence-nav-button.button-next').addClass('disabled').attr('aria-hidden', 'true').attr('disabled', 'disabled')
       return
 
     if @position == 1
-      @$('.sequence-nav-buttons .prev a').addClass('disabled').attr('aria-hidden', 'true')
+      @$('.sequence-nav-button.button-previous').addClass('disabled').attr('aria-hidden', 'true').attr('disabled', 'disabled')
```

Comment:
if we are using the HTML disabled attribute, we should be using aria-disabled=true, not aria-hidden